brayns/common/vocabulary/parameters.fbs

```diff
@@ -146,7 +146,7 @@ enum Engine: string {
 
 table Settings {
     engine: Engine;
-    timestamp: float;
+    animation_frame: uint;
```

Comment:
I think this one should be removed. It duplicates the frame event.

Reply:
Good point. Done